taxcalc/records.py

```diff
@@ -25,31 +32,178 @@ class Records(object):
                  **kwargs):
 
-        self.read(data)
-        self.read_blowup(blowup_factors)
-        self.read_weights(weights)
+        self.BF = read_blowup(blowup_factors)
+        self.WT = read_weights(weights)
+
+        attr = ['AGIR1', 'DSI', 'EFI', 'EIC', 'ELECT', 'e04600',
+                'FDED', 'FLPDYR', 'FLPDMO', 'f2441', 'e04250',
+                'f3800', 'f6251', 'f8582', 'f8606', 'IE',
+                'MARS', 'MIDR', 'n20', 'n24', 'n25', 'PREP',
+                'SCHB', 'SCHCF', 'SCHE', 'STATE', 'TFORM',
+                'TXST', 'XFPT', 'XFST', 'XOCAH', 'XOCAWH',
+                'XOODEP', 'XOPAR', 'XTOT', 'e00200', 'e00300',
+                'e00400', 'e00600', 'e00650', 'e00700', 'e00800',
+                'e00900', 'e01000', 'e01100', 'e01200', 'e01400',
+                'e01500', 'e01700', 'e02000', 'e02100', 'e02300',
+                'e02400', 'e02500', 'e03150', 'e03210', 'e03220',
+                'e03230', 'e03260', 'e03270', 'e03240', 'e03290',
+                'e03300', 'e03400', 'e03500', 'e00100', 'p04470',
+                'e04800', 'e05100', 'e05200', 'e05800', 'e06000',
+                'e06200', 'e06300', 'e09600', 'e07180', 'e07200',
+                'e07220', 'e07230', 'e07240', 'e07260', 'e07300',
+                'e07400', 'e07600', 'p08000', 'e07150', 'e06500',
+                'e08800', 'e09400', 'e09700', 'e09800', 'e09900',
+                'e10300', 'e10700', 'e10900', 'e59560', 'e59680',
+                'e59700', 'e59720', 'e11550', 'e11070', 'e11100',
+                'e11200', 'e11300', 'e11400', 'e11570', 'e11580',
+                'e11581', 'e11582', 'e11583', 'e10605', 'e11900',
+                'e12000', 'e12200', 'e17500', 'e18425', 'e18450',
+                'e18500', 'e19200', 'e19550', 'e19800', 'e20100',
+                'e19700', 'e20550', 'e20600', 'e20400', 'e20800',
+                'e20500', 'e21040', 'p22250', 'e22320', 'e22370',
+                'p23250', 'e24515', 'e24516', 'e24518', 'e24535',
+                'e24560', 'e24598', 'e24615', 'e24570', 'e25820',
+                'p25350', 'e25370', 'e25380', 'p25470', 'p25700',
+                'e25850', 'e25860', 'e25940', 'e25980', 'e25980',
+                'e25920', 'e25960', 'e26110', 'e26170', 'e26190',
+                'e26160', 'e26180', 'e26270', 'e26100', 'e26390',
+                'e26400', 'e27200', 'e30400', 'e30500', 'e32800',
+                'e33000', 'e53240', 'e53280', 'e53410', 'e53300',
+                'e53317', 'e53458', 'e58950', 'e58990', 'p60100',
+                'p61850', 'e60000', 'e62100', 'e62900', 'e62720',
+                'e62730', 'e62740', 'p65300', 'p65400', 'e68000',
+                'e82200', 't27800', 's27860', 'p27895', 'e87500',
+                'e87510', 'e87520', 'e87530', 'e87540', 'e87550',
+                'RECID', 's006', 's008', 's009', 'WSAMP', 'TXRT']
+
+        zero = ['e35300_0', 'e35600_0', 'e35910_0', 'x03150', 'e03600',
```

Comment:
same comment as above. The list here:
https://github.com/mmessick/Tax-Calculator/blob/luca/taxcalc/records.py#L682
is the same as this list.